gradle/run.gradle

```diff
@@ -43,9 +43,17 @@ testClusters {
       installedPlugins = Eval.me(installedPlugins)
       for (String p : installedPlugins) {
         plugin('plugins:'.concat(p))
+        if (p.equals("arrow-flight-rpc")) {
```

Comment:
@rishabhmaurya I think we discussed this in the past: these system properties will have an impact of others (notably, netty  based transports). Plus, they are in conflict of how we launch the OpenSearch from the standard distribution - see please https://github.com/opensearch-project/OpenSearch/blob/main/distribution/tools/launchers/src/main/java/org/opensearch/tools/launchers/SystemJvmOptions.java#L76

I would suggest to remove this changes and tackle them separately since we would need to look for possible conflicting settings and side effects.

Reply:
@reta sure, I will remove it. I kept them here since they would only be applied when this flight and transport are installed and will not impact any defaults.